book/src/k8s-test-runner/k8s-test-runner.md

```diff
@@ -130,6 +140,11 @@ Then:
         WASP_LOG_LEVEL = "info"
         TEST_LOG_LEVEL = "info"
         MERCURY_TEST_LOG_LEVEL = "info"
+        [metadata.labels]
+        "chain.link/component" = "k8s-test-runner"
```

Comment:
load-test
```suggestion
        "chain.link/component" = "load-test"
```
or something that identifies the kind/category of test it is might be useful so that if these kinds of test become ridiculously expensive, it is easy to identify